homeassistant/components/lightwave/__init__.py

```diff
@@ -88,7 +88,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         proxy_port = trv[CONF_PROXY_PORT]
         lwlink.set_trv_proxy(proxy_ip, proxy_port)
 
-        platforms = [CLIMATE_DOMAIN, SENSOR_DOMAIN]
+        platforms = PLATFORMS
         for platform in platforms:
```

Comment:
Remove intermediate variable `platforms`. Would be consistent with all the other integrations in this PR.

Reply:
Good spot. I've adjusted.